Error: Cannot find module 'gulp-util'

here s my task below:
````
let prelink= require('gulp-pre-link')
gulp.task('prelink', ()=>{
  return gulp.src('src/*.html)
  .pipe(prelink({}))
  .pipe(gulp.dest('build/'))
})
````
but is there any dependences that was dismissed caused i got an error msg like:
````
module.js:339
    throw err;
    ^

Error: Cannot find module 'gulp-util'
    at Function.Module._resolveFilename (module.js:337:15)
    at Function.Module._load (module.js:287:25)
    at Module.require (module.js:366:17)
    at require (module.js:385:17)
````
could u help with it~

--- a/package.json
+++ b/package.json
@@ -3,7 +3,9 @@
     "email": "openks@foxmail.com",
     "name": "朱阳星"
   },
-  "dependencies": {},
+  "dependencies": {
+    "gulp-util": "^3.0.7",
+  },
   "description": "replace link html with it's content ",
   "devDependencies": {},
   "directories": {},
@@ -13,6 +15,9 @@
   },
   "keywords": [
     "gulp"
+    ,"pre-link"
+    ,"gulp rel"
+    ,"gulp-pre-link"
   ],
   "license": "MIT",
   "main": "index.js",

